src/xbot/common/math/PIDManager.java

```diff
@@ -13,26 +13,37 @@
     private DoubleProperty propP;
     private DoubleProperty propI;
     private DoubleProperty propD;
+    private DoubleProperty maxOutput;
+    private DoubleProperty minOutput;
 
+    public PIDManager(String functionName, XPropertyManager propMan, double defaultP, double defaultI, double defaultD,
+            double defaultMaxOutput, double defaultMinOutput) {
+        setupPIDManager(functionName, propMan, defaultP, defaultI, defaultD, defaultMaxOutput, defaultMinOutput);
+    }
+    
     public PIDManager(String functionName, XPropertyManager propMan, double defaultP, double defaultI, double defaultD) {
-        setupPIDManager(functionName, propMan, defaultP, defaultI, defaultD);
+        setupPIDManager(functionName, propMan, defaultP, defaultI, defaultD, 1.0, -1.0);
     }
 
     public PIDManager(String functionName, XPropertyManager propMan) {
-        setupPIDManager(functionName, propMan, 0, 0, 0);
+        setupPIDManager(functionName, propMan, 0, 0, 0, 1.0, -1.0);
     }
 
     private void setupPIDManager(String functionName, XPropertyManager propMan, double defaultP, double defaultI,
```

Comment:
Why do we have this method? I think this should go in the most verbose constructor and then the others should just cascade down from least to most explicit.